File: R programming/Game_rock_paper_scissors.R
## Rock Scissors Paper
## Rock = 1, Paper = 2, Scissors = 3, Exit = 4


weapons <- c("rock", "paper", "scissors", "exit")
win <- 0
lose <- 0
tie <- 0

Rock_Scissors_Paper_Game <- function() {
  
  while (TRUE) {
    player_weapons <- as.numeric(readline("Choose your move: Rock[1], Paper[2], Scissors[3], Exit[4]: "))
    if (player_weapons == 4){
      cat("Good Bye My Friend!")
      break
    }
    
    player_weapons <- weapons[player_weapons]
    computer_weapons <- weapons[sample(1:3, 1)]
    
    if (player_weapons == computer_weapons) {
      tie <- tie + 1
      
    } else if (player_weapons == "paper" & computer_weapons == "scissors") {
      lose <- lose + 1
    } else if (player_weapons == "scissors" & computer_weapons == "rock") {
      lose <- lose + 1
    } else if (player_weapons == "rock" & computer_weapons == "papers") {
      lose <- lose + 1
    } else {
      win <- win + 1 
    } 
    
    cat("Player Weapons:", player_weapons, "\n")
    cat("Computer Weapons:", computer_weapons, "\n")
    cat("Win:",win, "lose:",lose,"Tie:",tie)
  }
}

File: Python programming/Game_rock_paper_scissors.py
import random as ran
def game():
    weapons = ["rock", "paper", "scissors"]
    win = 0
    lose = 0
    tie = 0
    while True:
          player_weapons = input("Choose you weapon: rock paper scissors: ")
          computer_weapons = ran.choice(weapons)

          if player_weapons == 'exit':
            print("Good bye my friend")
            break
          elif player_weapons == "paper" and computer_weapons == "scissors":
              print("You lose!")
              lose += 1
          elif player_weapons == "scissors" and computer_weapons == "rock":
              print("You lose!")
              lose += 1
          elif player_weapons == "rock" and computer_weapons == "paper":
              print("You lose!")
              lose += 1
          elif player_weapons == computer_weapons:
              print("You tie!")
              tie += 1
          else:
              print("You win!")
              win += 1
  
          print(f"Player Weapons : {player_weapons}")
          print(f"Computer Weapons : {computer_weapons}")
          print(f"Win: {win}, lose: {lose}, Tie: {tie}")
